libs/workspace-manager/src/lib/features/filesystem-navigator/filesystem-navigator-toolbar/filesystem-navigator-toolbar.component.ts

```diff
@@ -0,0 +1,37 @@
+import { NgForOf, NgIf } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  Output,
+} from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+
+@Component({
+  selector: 'cli-filesystem-navigator-toolbar',
+  standalone: true,
+  imports: [NgForOf, NgIf, MatButtonModule],
+  templateUrl: './filesystem-navigator-toolbar.component.html',
+  styleUrls: ['./filesystem-navigator-toolbar.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FilesystemNavigatorToolbarComponent implements OnChanges {
+  @Input() path: string | null = null;
+  @Output() pathChange = new EventEmitter<string>();
+
+  pathParts: string[] = [];
+
+  ngOnChanges(): void {
+    if (typeof this.path === 'string') {
+      this.pathParts = this.path.split('/').filter((v) => !!v);
```

Comment:
you should make sure it works correctly for windows as well, maybe even add a dedicated endpoint that returns the separator of the current platform (its a built-in in nodejs)

Reply:
will look into it